configure.php

```diff
@@ -0,0 +1,528 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Configure the alleyinteractive/create-wordpress-project interactively.
+ *
+ * This script will:
+ *
+ *   1. Prompt the user to replace the default values in the project.
+ *   2. Configure it for the project's hosting provider (VIP, Pantheon, etc.)
+ *   3. Scaffold default plugin for the project from alleyinteractive/create-wordpress-plugin.
+ *
+ * Supports arguments to set the values directly.
+ *
+ * [--project_name=<project_name>]
+ * : The project name.
+ *
+ * [--author_name=<author_name>]
+ * : The author name.
+ *
+ * [--author_email=<author_email>]
+ * : The author email.
+ *
+ * phpcs:disable
+ */
+
+namespace Create_WordPress_Project\Configure;
+
+if ( ! defined( 'STDIN' ) ) {
+	die( 'Not in CLI mode.' );
+}
+
+if ( 0 === strpos( strtoupper( PHP_OS ), 'WIN' ) ) {
+	die( 'Not supported in Windows. 🪟' );
```

Comment:
Are there parts of this script that are explicitly not supported in Windows, or did we just not test it there? Especially with new features in Windows like PowerShell and the Windows Subsystem for Linux (WSL) we might want to soften this to a warning and say that it might not work properly on Windows, but don't prevent a user from running it?

Reply:
I wouldn't mind an update to our `README.md` that says we've developed this to work on ____ operating systems, so your milage may vary on Windows - PRs welcome, yada yada